webapp/javascript/components/TimelineFlotChart.ts

```diff
@@ -0,0 +1,3706 @@
+/* eslint-disable */
```

Comment:
What is going on with this file? I assume you didn't write this code from scratch, so where is it coming from?

Reply:
Right, it's Flot's core rendering logic file. Changing appearance of timeline selection assumes editing code here. If you know any easier way please let me know.

Reply:
Alright, can you then
* point to the file this was copied from (and its version), probably just linking to the commit hash in github should be enough
* comment which sections you overwrote with custom logic

my perspective here is that anybody looking at this code 6 months from now will have a hard time understanding what's going on

Reply:
added comment on that file